UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/AccessibilityStatement/Index.cshtml

```diff
@@ -0,0 +1,49 @@
+    <ul class="msi-list-style">
+        <li><i class="bi bi-square-fill list-icon" aria-hidden="true"></i> Zoom in up to 200% without the text spilling off the screen.</li>
+        <li><i class="bi bi-square-fill list-icon" aria-hidden="true"></i> Navigate most of the website using just a keyboard.</li>
+        <li><i class="bi bi-square-fill list-icon" aria-hidden="true"></i> Navigate most of the website using speech recognition software.</li>
+        <li><i class="bi bi-square-fill list-icon" aria-hidden="true"></i> Listen to most of the website using a screen reader (including the most recent versions of NVDA and VoiceOver).</li>
+        <li><i class="bi bi-square-fill list-icon" aria-hidden="true"></i> See descriptive attributes for all images.</li>
+        <li><i class="bi bi-square-fill list-icon" aria-hidden="true"></i> See descriptions for all links.</li>
+    </ul>
+
+    <p>We’ve also made the website text as simple as possible to understand.</p>
+    <p><a href="https://abilitynet.org.uk/" target="_blank">AbilityNet</a> has advice on making your device easier to use if you have a disability.</p>
+    <h2>Feedback and contact information</h2>
+    <p>If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille:</p>
+    <ul class="msi-list-style">
+        <li><i class="bi bi-square-fill list-icon" aria-hidden="true"></i> Email <a href="mailto:customerservices@ukho.gov.uk">customerservices@ukho.gov.uk</a></li>
+        <li><i class="bi bi-square-fill list-icon" aria-hidden="true"></i> Call +44 (0)1823 484444</li>
+    </ul>
+
+    <p>We’ll consider your request and reply to you within 5 working days.</p>
+    <h3>Reporting accessibility problems with this website</h3>
+    <p>We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact <a href="mailto:customerservices@ukho.gov.uk">customerservices@ukho.gov.uk</a>to let us know your concerns.</p>
+    <h3>Enforcement procedure</h3>
+    <p>The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, <a href="https://www.equalityadvisoryservice.com/">contact the Equality Advisory and Support Service (EASS).</a>
+    </p>
+    <h3>Technical information about this website’s accessibility</h3>
+    <p>The UK Hydrographic Office is committed to making this website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.</p>
+    <h3>Compliance status</h3>
+    <p>This website is partially compliant with the Web Content Accessibility Guidelines version 2.1 AA standard, due to the non-compliances listed below.</p>
```

Comment:
We mention that there is a list of no-compliances below, but when I reach the section that is there to list them ( _Non-compliance with the accessibility regulations_ ), all I see is N/A.

Does that mean that the MSI service is absolutely compliant, or that we do not yet have a definitive list to write?

If it is the former, this sentence should be re-written to state that "This  website is wholly compliant with the Web Content Accessibility Guidelines version 2.1 AA standard." and the next two sections should be removed.

If it is that latter, do we have a task to update the list when they are known?

Reply:
As of now we do not have any non-compliant reasons and its work in progress.
We do have PBI https://dev.azure.com/ukhydro/Maritime%20Safety%20Information/_workitems/edit/52358 in backlog for updating accessibility statement page once all testing is done.

Reply:
Thank you, @usha13529, I'd missed that when I scanned the backlog 🙂

Reply:
After having a chat with @barrie-cooper and Brent, the best course of action is probably to leave it, as the update PBI is likely to be completed prior to the next release.